docs/getting-started-kubernetes.rst

```diff
@@ -1693,26 +1693,29 @@ Wait for NVIDIA DOCA Driver to install and apply the following CRs:
         k8s.v1.cni.cncf.io/resourceName: nvidia.com/mlnxnics
     spec:
       config: '{
-      "type": "ib-sriov",
       "cniVersion": "0.3.1",
       "name": "ib-sriov-network",
-      "metaPluginsConfigured": true,
-      "metaPlugins": {
-          "type": "rdma"
+      "plugins": [
+      {
+        "type": "ib-sriov",
+        "pkey": "0x6",
+        "link_state": "enable",
+        "ibKubernetesEnabled": true,
+        "ipam": {
+        "type": "whereabouts",
```

Comment:
we moved to NVIPAM need to be aligned with the other examples

Reply:
Thanks, fixed